sunpycube/cube/NDCube.py

```diff
@@ -0,0 +1,86 @@
+from sunpy.visualization.imageanimator import ImageAnimatorWCS
+import astropy.nddata
+
+__all__ = ['NDCube', 'Cube2D', 'Cube1D']
+
+
+class NDCube(astropy.nddata.NDData):
+    """docstring for NDCube"""
+
+    def __init__(self, data=None, wcs=None, **kwargs):
```

Comment:
this constructor should explicitly have all the options of the `NDData` one, so users can see what the options are in help etc.